### PR TITLE
Added Home Page

### PR DESCRIPTION
**Pull request by Amen-Musingarimi**
Added the Home page and added notes to the read-me file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```
Comment (ichala) [inline README.md]: - [ ] Great Job Creating Readme.MD file, but the template has been updated and you need to use the [new one](https://github.com/microverseinc/readme-template), kindly update it and make sure to fill in the necessary fields 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Amen-Musingarimi 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good .gitignore file  ✔️
- Linters Installed ✔️
- Correct Workflow ✔️


 ## Required Changes ♻️

- [ ] **_🛑 As part of the requirements the Pull Request should have a summary describing introduced changes in more detail. kindly update it since its empty right now_**
image

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Amen-Musingarimi) [reply README.md]: Hi @ichala , so after making the required changes am I supposed to make another pull request or the one that already made will still be working for the upcoming changes?

Comment (ichala) [reply README.md]: @Amen-Musingarimi keep working on this one , please do not close , merge or change the pull request until you get approved 🙏

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+# Authors - Amen-Musingarimi
+
+# client - html and css
+
+# Server - local server
+
+# Database - local
+
+# Key-Feature - home-page
```
Comment (Qoosim) [inline README.md]: - [ ] As requested by the previous reviewer, your README file does no still look prefessional yet. Check this [repo](https://github.com/Hassaanjbaig-code/Awersome-books) and learn how to customise README template to fit your project.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Amen-Musingarimi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  No linter errors. :heavy_check_mark: 
-  Descriptive PR title and summary. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Amen-Musingarimi, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**